### PR TITLE
Adding SwiftLint

### LINKED ISSUE #6: Add SwiftLint
**Code styling is important and should be uniform.**

Adding SwiftLint - whilst a pain in the ass at times - will eventually ensure that code is kept to a uniform standard, and that anybody reading through the code won't have to adjust to different authors.

### PR DESCRIPTION
**Pull request by mrianmerry**
### Proposed Changes

Fixes #6 _Add SwiftLint_
Added SwiftLint, resolved warnings, and cleaned up some other project stuff

* Added swiftlint build phase script - yaaay
* Added configuration file for swiftlint
* Cleaned up a load of violations
* Renamed `Run Script` build phases to something meaningful

### CODE REVIEW

File: Pokemon Tabletop Adventure/Data Models/WBPokémon.swift [hunk 1]
```diff
@@ -5,7 +5,7 @@
 //  Created by Ian Merry on 08/07/2018.
 //  Copyright © 2018 WeatherBear. All rights reserved.
 //
-
+// swiftlint:disable redundant_string_enum_value
```
Comment (d9rad) [inline Pokemon Tabletop Adventure/Data Models/WBPokémon.swift]: Already disabling it 🤣

File: Pokemon Tabletop Adventure/Data Models/WBPokémon.swift [hunk 2]
```diff
@@ -34,23 +35,23 @@ struct Pokémon: Codable {
...
     private enum CodingKeys: String, CodingKey {
         case dexNumber = "id"
-        case name
+        case name = "name"
         case stats = "base_stats"
-        case type
-        case abilities
-        case evolution
-        case biology
+        case type = "type"
```
Comment (d9rad) [inline Pokemon Tabletop Adventure/Data Models/WBPokémon.swift:46]: I don't think you need to give specific values unless it doesn't match the case

File: Pokemon Tabletop Adventure/Data Models/WBPokémon.swift [hunk 3]
```diff
@@ -72,69 +73,71 @@ struct PokeBiology: Codable {
     let hatchRate: Int?
     let diet: [DietType]
     let habitat: [Environment]
-    
+
     private enum CodingKeys: String, CodingKey {
-        case height
-        case weight
+        case height = "height"
```
Comment (d9rad) [inline Pokemon Tabletop Adventure/Data Models/WBPokémon.swift:78]: same for this

**Review by d9rad — DISMISSED** [review]
Looks good, just left a few comments

**Comment by d9rad** [comment]
@mrianmerry merge conflicts need resolving

File: Pokemon Tabletop Adventure/Data Models/WBPokémon.swift (hunk 2 above)
Comment (mrianmerry) [reply Pokemon Tabletop Adventure/Data Models/WBPokémon.swift:46]: That's what the swiftlint disable at the top was for - there are conflicting rules.
One was for preventing non-assigned enum values, and then that cunt up top for redundant assigned values.

File: Pokemon Tabletop Adventure/Data Models/WBPokémon.swift (hunk 1 above)
Comment (mrianmerry) [reply Pokemon Tabletop Adventure/Data Models/WBPokémon.swift]: Probably... I should just put this one into the disabled rules in the config?

**Review by d9rad — APPROVED** [review]

**Pull request merged**